src/panel/category_panel.js

```diff
@@ -50,24 +56,53 @@ export class CategoryPanel {
 
   }
 
+  async open (options = {}) {
+    SearchInput.minify()
+    document.querySelector('#panels').classList.add('panels--hide-services')
```

Comment:
This class is added in `open()` method but not removed in `close()` method, is this normal ? Idem for the `.map_bottom_button_group` in the `close()` method

Reply:
indeed we don't need this.